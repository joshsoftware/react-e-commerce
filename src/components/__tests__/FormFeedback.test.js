import { render, cleanup } from '@testing-library/react';
import React from 'react';
import { FormFeedback } from 'reactstrap';

afterEach(cleanup);

describe('FormCard Component', () => {
  it('Must return FormCard Component', () => {
    const { asFragment } = render(<FormFeedback />);
    expect(asFragment()).toMatchSnapshot();
  });
});
