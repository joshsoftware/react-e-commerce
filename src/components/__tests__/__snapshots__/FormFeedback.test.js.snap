// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCard Component Must return FormCard Component 1`] = `
<DocumentFragment>
  <div
    class="invalid-feedback"
  />
</DocumentFragment>
`;
